package.json

```diff
@@ -99,5 +99,8 @@
     },
     "publishConfig": {
         "tag": "experimental"
+    },
+    "dependencies": {
+        "ilib-resbundler": "^1.0.1"
```

Comment:
ilib should not have any direct dependencies

Reply:
yes, it's shouldn't be there. I guess it's accidentally added. I updated it.